tests/e2e/Services/Databases/DatabasesCustomServerTest.php

```diff
@@ -593,6 +593,200 @@ public function testUpdateCollection(array $data)
+        ]), [
+            'key' => 'age',
+            'min' => 0,
+            'max' => 120,
+            'required' => false,
+        ]);
+
+        $alias = $this->client->call(Client::METHOD_POST, $attributesPath . '/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'alias',
+            'size' => 256,
+            'required' => false,
+        ]);
+
+     /**
+      * Check Status of every Attribute
+      */
+        $this->assertEquals(202, $firstName['headers']['status-code']);
+        $this->assertEquals('firstName', $firstName['body']['key']);
+        $this->assertEquals('string', $firstName['body']['type']);
+
+        $this->assertEquals(202, $lastName['headers']['status-code']);
+        $this->assertEquals('lastName', $lastName['body']['key']);
+        $this->assertEquals('string', $lastName['body']['type']);
+
+        $this->assertEquals(202, $alias['headers']['status-code']);
+        $this->assertEquals('alias', $alias['body']['key']);
+        $this->assertEquals('string', $alias['body']['type']);
+
+        $this->assertEquals(202, $age['headers']['status-code']);
+        $this->assertEquals('age', $age['body']['key']);
+        $this->assertEquals('integer', $age['body']['type']);
+
+        // Wait for database worker to finish creating attributes
+        sleep(5);
+
+        // Creating document to ensure cache is purged on schema change
+        $document = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $actors['body']['$id'] . '/documents', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'documentId' => ID::unique(),
+            'data' => [
+                'firstName' => 'Jonah',
+                'lastName' => 'Jameson',
+                'age' => 25,
+                'alias' => 'JJ'
+            ],
+            'permissions' => [
+                Permission::read(Role::any()),
+                Permission::update(Role::any()),
+                Permission::delete(Role::any()),
+            ],
+        ]);
+
+        $index = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $actors['body']['$id'] . '/indexes', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'key_lastName',
+            'type' => 'key',
+            'attributes' => [
+                'lastName',
+            ],
+        ]);
+
+        // Wait for database worker to finish creating index
+        sleep(2);
```

Comment:
Let's remove this, doesn't make much sense to add an index on an encrypted attribute since it can't be queried